From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Wed, 22 Jun 2011 00:31:47 +0200
Subject: [PATCH] set the escape delay with set_escdelay

This fixes building with ncurses-5.9:
driver/ncurses.c: In function 'ncurses_init_graphics':
driver/ncurses.c:267:5: error: lvalue required as left operand of assignment

Origin: upstream, http://caca.zoy.org/changeset/4643/libcaca/trunk/caca/driver/ncurses.c
Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 caca/driver/ncurses.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/caca/driver/ncurses.c b/caca/driver/ncurses.c
index 274cdd8..4fe950c 100644
--- a/caca/driver/ncurses.c
+++ b/caca/driver/ncurses.c
@@ -264,7 +264,7 @@ static int ncurses_init_graphics(caca_display_t *dp)
     mouseinterval(-1); /* No click emulation */
 
     /* Set the escape delay to a ridiculously low value */
-    ESCDELAY = 10;
+    set_escdelay(10);
 
     /* Activate colour */
     start_color();
